integration-test-rex/pom.xml

```diff
@@ -0,0 +1,557 @@
+                      </command>
+                      <command>
+                        /subsystem=logging/console-handler=CONSOLE:change-log-level(level=DEBUG)
+                      </command>
+                      <command>
+                        /subsystem=logging/root-logger=ROOT:change-root-log-level(level=DEBUG)
+                      </command>
+
+                      <!-- Add microprofile stuff -->
+                      <command>
+                        /extension=org.wildfly.extension.microprofile.config-smallrye:add
+                      </command>
+                      <command>
+                        /subsystem=microprofile-config-smallrye:add
+                      </command>
+                      <command>
+                        /extension=org.wildfly.extension.microprofile.fault-tolerance-smallrye:add
+                      </command>
+                      <command>
+                        /subsystem=microprofile-fault-tolerance-smallrye:add
+                      </command>
+                      <command>
+                        /extension=org.wildfly.extension.microprofile.metrics-smallrye:add
+                      </command>
+                      <command>
+                        /subsystem=microprofile-metrics-smallrye:add
+                      </command>
+                      <!--log sql queries and parameter bindings-->
+                      <!--command>
+                        /subsystem=logging/logger=org.hibernate.SQL:add(level=DEBUG)
+                      </command>
+                      <command>
+                        /subsystem=logging/logger=org.hibernate.type:add(level=TRACE)
+                      </command-->
+                      <command>
+                        /subsystem=security/security-domain=keycloak/:add
+                      </command>
+                      <command>
+                        /subsystem=security/security-domain=keycloak/authentication=classic/:add(login-modules=[{ "code" => "org.keycloak.adapters.jboss.KeycloakLoginModule","flag" => "required"}])
+                      </command>
+                      <command>
+                        /extension=org.keycloak.keycloak-adapter-subsystem/:add(module=org.keycloak.keycloak-adapter-subsystem)
+                      </command>
+                      <command>
+                        /subsystem=keycloak:add
+                      </command>
+                      <!--command>
+                        /subsystem=keycloak/realm=newcastle-testcontainer:add(realm-public-key="MIIBIjAN...", auth-server-url=" ", ssl-required="none", disable-trust-manager="true")
+                      </command-->
+                      <command> <!-- TODO realm-public-key. Do we need it ? -->
+                        /subsystem=keycloak/realm=newcastle-testcontainer:add(auth-server-url="http://localhost:${keycloakPort}", ssl-required="none", disable-trust-manager="true")
```

Comment:
this should be the fixed port right? is the used keycloakPort correct at this point?

Reply:
It's dynamic, taking a free port.